code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -732,10 +753,12 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	if(id)
 		remove_id()
-		playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/terminal_eject_disc.ogg', 50, 1)
 	else
 		remove_pen()
-		playsound(src, 'sound/machines/button4.ogg', 50, 1)
+		if (!silent)
+			playsound(src, 'sound/machines/button4.ogg', 50, 1)

```

Comment:
This is a mechanical sound, making this mutable via a software toggle makes no logical sense